crates/oxc_linter/src/config/config_builder.rs

```diff
@@ -908,49 +920,149 @@ mod test {
 
     #[test]
     fn test_extends_plugins() {
+        use std::fs;
+
+        // Create a temporary directory for test files
+        let temp_dir = tempfile::tempdir().unwrap();
+        let parent_path = temp_dir.path().join("parent.json");
+        let child_no_plugins_path = temp_dir.path().join("child_no_plugins.json");
+        let child_with_plugins_path = temp_dir.path().join("child_with_plugins.json");
```

Comment:
Let's try to avoid adding a dependency unless we really need it, so I would recommend creating some additional files in the `fixtures` directory that have the content specified here.

Reply:
Thanks @camchenry I've just addressed this